start.js

```diff
@@ -0,0 +1,6 @@
+const server = require('./server.js');
+const http = require('http');
+
+http.createServer(server).listen(server.get ('port'), function (){
```

Comment:
you don't need to do this, with express you can just do `server.listen(...`

see: https://expressjs.com/en/starter/hello-world.html